open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/server/mappers/entities/EntityDetailMapper.java

```diff
@@ -125,15 +125,14 @@ private void mapEntityDetailPropertiesToNode(N node, InstanceProperties omrsEnti
                         while (iter.hasNext()) {
                             String mapkey = iter.next();
                             PrimitivePropertyValue primitivePropertyMapValue = (PrimitivePropertyValue) instancePropertyForMap.getPropertyValue(mapkey);
-                            String mapvalue =  primitivePropertyMapValue.getPrimitiveValue().toString();
+                            String mapvalue = primitivePropertyMapValue.getPrimitiveValue().toString();
                             actualMap.put(mapkey, mapvalue);
                         }
                         node.setAdditionalProperties(actualMap);
                     } if (!mapMapToNode(node, propertyName, instancePropertyForMap)) {
                         if (null==node.getAdditionalProperties())  {
                             node.setAdditionalProperties(new HashMap<String, String>());
                         }
-                        node.getAdditionalProperties().put(propertyName,mapPropertyValue.valueAsString());
```

Comment:
I think we are saying with this change that Additional properties are supported using a property called additionalProperties that is of type map and will be stored as a String string map. If this isthe case this change should remove the previous lines if, and the enclosing if. Also the same code needs to be removed from the enum case in this method.

Reply:
@davidradl Yes, I removed these.